pkgs/top-level/all-packages.nix

```diff
@@ -7187,8 +7187,6 @@ in
 
   smu = callPackage ../tools/text/smu { };
 
-  smugline = python3Packages.smugline;
```

Comment:
for top level attrs, we usually add an entry to `pkgs/top-level/aliases.nix` throwing an error stating why it was removed.

Reply:
OK, thank you for advice!

Reply:
Done.